_docs/_latest/setup/manual-gcp-deployment.md

```diff
@@ -0,0 +1,117 @@
+It is highly recommended that you use the [automated installer](({% link _docs/latest/setup/install.md %})
+for an easy and error-free deployment.
+
+But in case you prefer to install and deploy Forseti on your own, or just
+curious what goes into the installation process, you can follow the steps below.
+
+These steps assume you have certain level of GCP knowledge (how to use gcloud,
+where to bind the service account to the VM in the Cloud Console, etc),
+and will by necessity point to the installer and Deployment Templates for
+specific details of the commands to use.
+
+# Create Project
```

Comment:
The style looks a bit weird for "Create Project" all the way to "Assign a billing account" since there is no content between them, should we change that up a bit? (maybe adding some content)

Reply:
Done